app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -81,8 +86,13 @@ public DaoSession getDaoSession() {
 
     /**
      * Initializes Retrofit and makes it available to all activities.
+     *
+     * @param progressUpdateCallback If the request needs to update progress to user (in case of downloading or
+     *                               uploading
```

Comment:
Will the app be _uploading_ files?

Reply:
In case we have to uploading a big file in the future, then we also can use this ProgressUpdateCallback for uploading progress.

Reply:
Yes sure.